lib/stack_master/parameter_resolvers/ejson.rb

```diff
@@ -27,9 +28,12 @@ def validate_ejson_file_specified
       end
 
       def decrypt_ejson_file
-        @decrypt_ejson_file ||= EJSONWrapper.decrypt(ejson_file_path,
-                                                     use_kms: @stack_definition.ejson_file_kms,
-                                                     region: ejson_file_region)
+        ejson_file_key = credentials_key
```

Comment:
@stevehodgkiss Is my assumption that the scope for the cached decrypted secrets is the credentials?

Reply:
Yeah, I think that assumption is correct. The file can't change between parameters right now (it's in stack definition / stack_master.yml config).

Reply:
That's my understanding too. Thanks for confirming.